sql/modules/Reconciliation.sql

```diff
@@ -101,9 +147,10 @@ $$
                                     and approved IS NOT TRUE);
     DELETE FROM cr_report
      WHERE id = in_report_id AND entered_username = SESSION_USER
-           AND submitted IS NOT TRUE AND approved IS NOT TRUE
-    RETURNING TRUE;
-$$ LANGUAGE SQL SECURITY DEFINER;
+           AND submitted IS NOT TRUE AND approved IS NOT TRUE;
+    RETURN FOUND;
+END;
+$$ LANGUAGE PLPGSQL SECURITY DEFINER;
```

Comment:
Why do you want to change the procedure to PLPGSQL? The two conditions I see can't occur, or can be integrated.

Reply:
FOUND requires PLPGSQL

Reply:
So does `RAISE EXCEPTION`; my point is: do we really need to raise an exception? Or simply add conditions to the DELETE statements which make them "update nothing when the preconditions are not met"?

The function used to be defined to return FALSE on failure. You've made it raise an exception on failure. That's a major change in paradigm. Is that required, or can we do with a smaller change?